examples/dask.toml

```diff
@@ -41,3 +41,4 @@ exclude = [ "dask.utils:Dispatch",
 
 #docs_path = "~/dev/dask/docs/source"
 exec_failure = 'fallback'
+execute_doctests = false
```

Comment:
You introduced this but are not using it, I'm guessing you intended something to `exec`. I pushed a commit that rename all the usage of `config.exec` to `config.execute_doctests` for it to work as intended. You naming is better.